tasks.php

```diff
@@ -1602,20 +1597,16 @@ function NotificationMail($tid, $message, $new_task = false)
 function RelatedTasks($tid)
 {
     global $tasks_url, $tasks_status_array;
-    $result = wrapped_mysql_query("SELECT related_tasks FROM tasks WHERE task_id = $tid");
-    $row = mysqli_fetch_assoc($result);
-    $related_tasks = $row["related_tasks"];
     echo "<table class='tasks'>\n";
     echo "<tr><td style='width: 100%'><b>Related Tasks&nbsp;&nbsp;</b>";
     echo "<form action='$tasks_url' method='post'>";
     echo "<input type='hidden' name='action' value='add_related_task'>";
     echo "<input type='hidden' name='task_id' value='$tid'>";
     echo "<input type='number' name='related_task' min='1' required>&nbsp;&nbsp;";
     echo "<input type='submit' value='Add'>\n";
-    echo " (Add the number of an existing, related task. This is optional.)";
+    echo " (Add the number of an existing, related task.)";
```

Comment:
Removing "optional" concerns me -- we don't want to imply that a related task is required.